src/components/Chat/ChatItem.tsx

```diff
@@ -63,12 +63,13 @@ class ChatItem extends UIComponent<ReactProps<ChatItemProps>, any> {
 
   private renderChatItem(styles: ComponentSlotStylesPrepared) {
     const { message, gutter, gutterPosition } = this.props
-    const gutterElement = gutter && Slot.create(gutter, { defaultProps: { styles: styles.gutter } })
+    const gutterElement =
+      gutter && Component.create(gutter, { defaultProps: { styles: styles.gutter } })
```

Comment:
No need to short circuit shorthand factories, they will gracefully handle null/undefined shorthand values.

```suggest
Component.create(gutter, { defaultProps: { styles: { styles.gutter } })
```

Reply:
unfortunately, there was a bug there that did not short circuit the creation of elements; I must have forgot to create an issue for it; I'll check again